dojo/db_migrations/0047_jira_minimum_severity_default.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.14 on 2020-07-10 08:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    def set_default_to_low(apps, schema_editor):
+        system_settings = apps.get_model('dojo', 'System_Settings')
```

Comment:
does this work? I would expect something like 
```
System_Settings = apps.get_model('dojo', 'System_Settings')
ss = System_Settings.objects.get(0)
...
```

But maybe the thing in the PR also works?